### ISSUE TITLE
Do not remove tags inside comments

### ISSUE THREAD

**Issue by Artur-** [Reviewable/Reviewable#263]
```
A: This should not be a <div>, it should be something else
B: Yes, how about <span>
B: Or maybe <b>
```

In reviewable, this discussion becomes a bit hard to follow as you see
```
A: This should not be a , it should be something else
B: Yes, how about
B: Or maybe
```

**Comment by pkaminski** [comment]
Markdown allows (some) inline HTML -- this is a feature, not a bug.  You'll find that both GitHub and Reviewable exhibit the same behavior on the example above, because Reviewable actually uses GitHub's Markdown rendering function.  Like so:

```
Yes, how about <span>
```
renders as -> Yes, how about <span>

Instead, you need to code-escape the tag:

```
Yes, how about `<span>`
```
renders as -> Yes, how about `<span>`
